public/favicons/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
is this file necessary for all browsers also?
manifest.json is  for responsive web, isn't it?